css/style.css

```diff
@@ -1,4 +1,503 @@
+}
+
+.menubar__link_red:nth-child(1) {
+  margin-right: 63px;
+}
+
+.menubar__link:nth-child(2) {
+  margin-right: 25px;
+}
+
+.menubar__link:nth-child(3) {
+  margin-right: 44px;
+}
+
+.menubar__link:nth-child(4) {
+  margin-right: 48px;
+}
+
+.menubar__link:nth-child(5) {
+  margin-right: 221px;
+}
+
+.menubar__link:nth-child(6) {
+  margin-right: 32px;
+  font-size: 16px;
+}
+
+.menubar__link:nth-child(7) {
+  margin-right: 32px;
+  font-size: 16px;
+}
+
+.button {
+  display: block;
+  background-color: #fff;
+  width: 24px;
+  height: 16px;
+  position: relative;
+  transition: all 0.5s ease 0s;
+}
+
+.button:nth-child(8) {
+  margin-right: 10px;
+}
+
+.button__line {
+  position: absolute;
+  width: inherit;
+  height: 2px;
+  background-color: #e51937;
+}
+
+.second {
+  top: 7px;
+}
+
+.third {
+  top: 14px;
+}
+
+.button:hover, 
+.button:focus, 
+.button:active {
+  transform: rotate(90deg);
+  cursor: pointer;
+  transition: all 0.5s ease 0s;
+}
+
+.content {
+  position: relative;
+}
+
+.content__main-title, 
+.content__title {
+  text-align: center;
+  margin: 0;
+}
+
+.content__main-title {
+  font-weight: 600;
+  font-size: 90px;
+  margin-bottom: -3px;
+}
+
+.content__title {
+  font-weight: 600;
+  font-size: 16px;
+}
+
+.main-tesla {
+  position: relative;
+}
+
+.tesla-bgd {
+  position: absolute;
+  top: -119px;
+  left: -45px;
+}
+.tesla-car {
+  width: 968px;
+  height: 450px;
+  padding: 24px 0 20px 154px;
+}
+
+.content__data-car {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 256px 22px 270px;
+}
+
+.content__facts {
+  display: flex;
+  margin-right: 81px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 3px 0 0 3px;
+}
+
+.data__speed-icon {
+  width: 54px;
+  height: 42px;
+  padding-right: 10px;
+}
+
+.content__road-icon {
+  width: 40px;
+  height: 59px;
+  padding-right: 7px;
+  padding-top: 7px;
+}
+
+.data__speed-block {
+  padding-right: 82px;
+}
+
+.data__road-block {
+padding-bottom: 4px;
+}
+
+.data__dimention {
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: flex-end;
+  
+}
+
+.data__figures {
+  font-weight: 600;
+  font-size: 35px;
+  padding-right: 3px;
+}
+
+.data__speed {
+  font-weight: 600;
+  font-size: 21px;
+}
+
+.data__specifications {
+  display: inline-block;
+  font-size: 13px;
+}
+
+.test-btn {
+  display: flex;
+  padding-right: 19px;
+}
+
+.testdrive {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  white-space: nowrap;
+  width: 118px;
+  margin-right: 40px;
+}
+
+.testdrive__img {
+  width: 22px;
+  height: 22px;
+  padding-right: 6px;
+}
+
+.testdrive__text {
+  font-weight: bold;
+  font-size: 14px;
+  text-transform: uppercase;
+}
+
+.order-btn {
+  border: 2px solid #e51937;
+  width: 160px;
+  height: 40px;
+  background-color: #fff;
+  font-weight: bold;
+  font-size: 14px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #e51937;
+  transition: all 0.5s ease 0s; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5652830)